internal/integration_test/vs/wasmedge/wasmedge.go

```diff
@@ -1,4 +1,4 @@
-//go:build amd64 && cgo && !windows && wasmedge
```

Comment:
nice to have another exception gone. I verified the lib exists even if it is pinned to specific macos